tests/modules/text_generation/test_text_generation_tgis.py

```diff
@@ -9,21 +9,36 @@
 import pytest
 
 # Local
-from caikit_nlp.modules.text_generation import TextGeneration
+from caikit_nlp.modules.text_generation import TextGenerationTGIS
 from tests.fixtures import (
     CAUSAL_LM_MODEL,
     SEQ2SEQ_LM_MODEL,
     StubTGISBackend,
     StubTGISClient,
 )
 
+### Stub Modules
+
+# Helper stubs / mocks; we use these to patch caikit so that we don't actually
+# test the TGIS backend directly, and instead stub the client and inspect the
+# args that we pass to it.
+class StubClient:
```

Comment:
Looks like this is unused - I assume this is from StubTGISClient duplication before it was moved into the common fixture. Can we remove it?

Reply:
yep. seems like leftover from rebase. removing it